scripts/fetch-schema.js

```diff
@@ -10,7 +10,7 @@ const thegraphConfig = require('../graphql.thegraph.config')
 const exec = promisify(child_process.exec)
 
 function fetchSchema(url, outputFile) {
-  exec(`npx get-graphql-schema --h Origin=https://app.uniswap.org ${url}`)
+  exec(`get-graphql-schema --h Origin=https://app.uniswap.org ${url}`)
```

Comment:
When left in the use of `npx` causes an error as the machine's `npm` version is out of date. We currently do not specify npm version in our package.json `engines` to suggest use of `yarn`. Replacing `npx` with the following `yarn`, `yarn x`, `yarn dlx`, `yarn create`, all lead to syntax errors when trying to parse the retrieved schema.